doc/releases/changelog-0.39.0.md

```diff
@@ -4,227 +4,454 @@
+  * `qml.spin.spin_hamiltonian`: generates a spin `Hamiltonian` object given a `Lattice` object with custom edges/nodes.
+
+  An example is shown below for a :math:`3 \times 3` triangular lattice with open boundary conditions.
+
+  ```python
+  lattice = qml.spin.Lattice(
+      n_cells=[3, 3],
+      vectors=[[1, 0], [np.cos(np.pi/3), np.sin(np.pi/3)]],
+      positions=[[0, 0]],
+      boundary_condition=False
+  )
+  ```
+
+  We can validate this `lattice` against `qml.spin.generate_lattice('triangle', ...)` by checking the 
+  `lattice_points` (the :math:`(x, y)` coordinates of all sites in the lattice):
+
+  ```pycon
+  >>> lp = lattice.lattice_points
+  >>> triangular_lattice = qml.spin.generate_lattice('triangle', n_cells=[3, 3])
+  >>> np.allclose(lp, triangular_lattice.lattice_points)
+  True
+  ```
+
+  The `edges` of the `Lattice` object are nearest-neighbour by default, where we can add edges by using
+  its `add_edge` method. 
+
+  Optionally, a `Lattice` object can have interactions and fields endowed to it by specifying values 
+  for its `custom_edges` and `custom_nodes` keyword arguments. The Hamiltonian can then be extracted
+  with the `qml.spin.spin_hamiltonian` function. An example is shown below for the transverse-field 
+  Ising model Hamiltonian on a :math:`3 \times 3` triangular lattice. Note that the `custom_edges` and 
+  `custom_nodes` keyword arguments only need to be defined for one unit cell repetition.
+
+  ```python
+  edges = [
+      (0, 1), (0, 3), (1, 3)
+  ]
+
+  lattice = qml.spin.Lattice(
+      n_cells=[3, 3],
+      vectors=[[1, 0], [np.cos(np.pi/3), np.sin(np.pi/3)]],
+      positions=[[0, 0]],
+      boundary_condition=False,
+      custom_edges=[[edge, ("ZZ", -1.0)] for edge in edges], 
+      custom_nodes=[[i, ("X", -0.5)] for i in range(3*3)],
+  )
+  ```
+
+  ```pycon
+  >>> tfim_ham = qml.spin.transverse_ising('triangle', [3, 3], coupling=1.0, h=0.5)
+  >>> tfim_ham == qml.spin.spin_hamiltonian(lattice=lattice)
+  True
+  ```
+
+* More industry-standard spin Hamiltonians have been added in the `qml.spin` module.
+  [(#6174)](https://github.com/PennyLaneAI/pennylane/pull/6174)
+  [(#6201)](https://github.com/PennyLaneAI/pennylane/pull/6201)
 
-<h4>Spin Hamiltonians 💞</h4>
+  Three new industry-standard spin Hamiltonians are now available with PennyLane v0.39:
 
-* Function is added for generating the spin Hamiltonian for the
-  [Kitaev](https://arxiv.org/abs/cond-mat/0506438) model on a lattice.
-  [(#6174)](https://github.com/PennyLaneAI/pennylane/pull/6174)
+  * `qml.spin.emery` : the [Emery model](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.58.2794)
+  * `qml.spin.haldane` : the [Haldane model](https://journals.aps.org/prl/pdf/10.1103/PhysRevLett.61.2015)
+  * `qml.spin.kitaev` : the [Kitaev model](https://arxiv.org/abs/cond-mat/0506438)
```

Comment:
@josh146 FYI, if there is a backtick next to a colon, sphinx seems to think you're indicating that as a directive, which it then has no idea about lol. Putting a space in between the backtick and the colon fixes it.